src/content/docs/en/reference/programmatic-reference.mdx

```diff
@@ -246,4 +277,59 @@ import { sync } from "astro";
 await sync({
   root: "./my-project",
 });
-```
\ No newline at end of file
+```
+
+## `mergeConfig()`
+
+<p>
+
+**Type:** `<T extends AstroConfig | AstroInlineConfig>(config: T, overrides: DeepPartial<T>) => T`
+</p>
+
+Imported from `astro/config`, merges a partial Astro configuration on top of an existing, valid, Astro configuration.
+
```

Comment:
```suggestion

The merge behaves according to the following rules:

```
Do we need some kind of sentence to introduce the bullet points maybe? (And, maybe that's not the right sentence -- just off the top of my head. It seems like we should set some context for "list of bullet points.")